README.md

```diff
@@ -1 +1,105 @@
-# SproutSpy-3.0
\ No newline at end of file
+# Sprout Spy 3.0
+
+--- [![Deploy](https://www.herokucdn.com/deploy/button.svg)] --- ADD NEW LINK TO DEPLOYED APP
+
+## Table of contents
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [Screenshot](#Screenshot)
+* [Setup](#setup)
+* [License](#License)
+* [Contact](#Contact)
+
+
+## General info
+
+Our project, "Sprout Spy", is an application that will help new or future gardeners understand what kinds of seeds and cuttings can be planted in their local
+planting zone, as well as search for local public gardens for those who do not have planting space in their area. Our application will also display the day's current weather to plan your trip to your local garden accordingly.
+
+
+<ins>User Story</ins>
+
+```
+AS A future gardener
+I WANT: -to know what plants I can grow near me,
+        -to get instructions for growing those plants
+        -to find a community garden
+        -to know the current weather to plan the day accordingly
+        -to sign up as a new user or login, in order to add a plant to the database
+SO THAT: I can start gardening with confidence
+```
+<ins>Acceptance Criteria</ins>
+
+```
+GIVEN I am using the Sprout Spy website to learn how to garden, 
+WHEN I open the Sprout Spy website, 
+THEN i see a a welcome modal that lightens the mood of anxious future gardeners.
+WHEN I click an answer for the modal, 
+THEN I see the Openweathermaps header, and google maps API to search for local gardens and local planting zone information.
+WHEN I enter a zip code in the search bar,
+THEN I see with the results for the corresponding agricultural zone as well as local gardens.
+WHEN I click to see results,
+THEN I am taken to the results page that lists plants I can grow in my zone.
+WHEN I log in or sign up,
+THEN I can add a plant to the database.
+```
+	
+## Technologies
+This project is created with:
+** Html
+* CSS
+* Javascript
+* Jquery
+* Google Maps API
+* Google Places API
+* C0bra API
+* Openweathermaps API
+* Full MERN Stack
+
+--- UPDATE TECH ---
```

Comment:
right on chris! couple of updates is all we need and we got it; since we're utilizing the MERN stack we can actually get rid of our CSS, Javascript, and 
 HTML, bullets in our technologies section..we are here bro! :)